test/ignore-errors.ts

```diff
@@ -700,89 +708,89 @@ describe('when setting ignoreErrors=true', function () {
         await updateWorkItem(this.backend, workItem);
         const currentWorkItems = (await getWorkItemsByJobId(db, workItem.jobID)).workItems;
         expect(currentWorkItems.length).to.equal(4);
-        expect(currentWorkItems.filter((item) => item.status === WorkItemStatus.READY && item.serviceID === 'sds/swot-reproject:latest').length).to.equal(3);
+        expect(currentWorkItems.filter((item) => item.status === WorkItemStatus.READY && item.serviceID === 'ghcr.io/podaac/l2ss-py:sit').length).to.equal(3);
       });
 
-      describe('when the first swot-reprojection service work item fails', function () {
-        let firstSwotItem;
+      describe('when the first L2-Subsetter service work item fails', function () {
+        let firstL2SSItem;
 
         before(async function () {
           let shouldLoop = true;
           // retrieve and fail work items until one exceeds the retry limit and actually gets marked as failed
           while (shouldLoop) {
-            const res = await getWorkForService(this.backend, 'sds/swot-reproject:latest');
-            firstSwotItem = JSON.parse(res.text).workItem;
-            firstSwotItem.status = WorkItemStatus.FAILED;
-            firstSwotItem.results = [];
+            const res = await getWorkForService(this.backend, 'ghcr.io/podaac/l2ss-py:sit');
```

Comment:
Should the harmony tests be using the SIT version of this? Is that because it's not in PROD yet?

Reply:
I mimicked this from `test/concatenation.ts`, where the PO.DAAC SIT images are used. I've not plumbed the depths to find exactly where the available services are defined, so I'm not sure if a production-tagged image would also work.